_docs/tasks/metrics-logs.md

```diff
@@ -9,210 +10,300 @@ type: markdown
 ---
 {% include home.html %}
 
-This task shows how to configure Mixer to automatically gather telemetry
-for a service within a cluster. At the end of this task, a new metric and
-a new log stream will be enabled for calls to a specific service within your
-cluster.
+This task shows how to configure Istio to automatically gather telemetry for
+services in a mesh. At the end of this task, a new metric and a new log stream
+will be enabled for calls to services within your mesh.
 
 The [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application is used
 as the example application throughout this task.
 
 ## Before you begin
-* [Install Istio](./installing-istio.html) in your kubernetes
-  cluster and deploy an application.
-
-* Configure your environment to support calling `istioctl mixer`.
-  This may require setting up port-forwarding for the Mixer Config API as described in the
-  [reference docs]({{home}}/docs/reference/commands/istioctl.html#istioctl-mixer) for `istioctl mixer`.
+* [Install Istio](./installing-istio.html) in your kubernetes cluster and deploy
+  an application. This task assumes that Mixer is setup in a default
+  configuration (`--configDefaultNamespace=istio-config-default`). If a
+  different value is used, the configuration in this Task (and commands to be
+  issued) must be updated to match.
 
-* Configure your environment to support accessing the Istio dashboard, as described in the 
-  [Installation Guide](./installing-istio.html). This requires installing the optional add-ons
-  ([Prometheus](https://prometheus.io) and [Grafana](https://grafana.com/)), as well as verifying access to
-  the dashboard. The Istio dashboard will be used to verify task success.
+* Install the optional add-on [Prometheus](https://prometheus.io). Prometheus
```

Comment:
Can we use this opportunity to move that content from install task?

Reply:
As discussed offline, i'm going to create a "Using Addons" or similar task.